packages/react/src/ui-kits/priceBanner/index.tsx

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
 import { CurrencySymbols } from '@heliofi/sdk';
 
-import { StyledPriceBanner, StyledTitle, StyledPrice } from './styles';
+import {
+  StyledPriceBanner,
+  StyledTitle,
+  StyledPrice,
+  StyledPriceBannerWrapper,
+  StyledPriceBannerLabel,
+} from './styles';
 
 export type PriceBannerProps = {
+  label?: string;
   title?: string;
   amount: number;
   currency?: string;
 };
 
-const PriceBanner = ({ title, amount, currency }: PriceBannerProps) => (
-  <StyledPriceBanner>
-    {title && <StyledTitle>{title}</StyledTitle>}{' '}
-    <StyledPrice>
-      {currency === CurrencySymbols.USDC ? '$' : ''}
-      {amount} {currency}
-    </StyledPrice>
-  </StyledPriceBanner>
+const PriceBanner = ({ label, title, amount, currency }: PriceBannerProps) => (
```

Comment:
amount is number here, will this cause an issue ? is it decimalAmount or minimalUnitAmount

Reply:
this is visual price @martinavagyan . it's already converted

Reply:
ok so it's decimalAmount, if you see its not a lot of impact I would suggest to update the name